src/openrct2/platform/Platform.Posix.cpp

```diff
@@ -59,6 +60,32 @@ namespace Platform
         return std::string(value, colon);
     }
 
+    std::string GetOsName()
+    {
+        struct utsname buffer;
+
+        errno = 0;
+        if (uname(&buffer) < 0)
+        {
+            log_error("Couldn't get OS");
```

Comment:
If it failed, would be great to also know why: at least provide `errno`

Reply:
Added Console::WriteLine(errno)